app/src/main/java/com/github/polypoly/app/menu/WelcomeActivity.kt

```diff
@@ -138,13 +140,21 @@ class WelcomeActivity : ComponentActivity() {
         ) {
             ProfileOptionButton()
             RulesOptionButton()
-            OptionButton(
-                onClick = { /*TODO*/ },
-                icon_id = R.drawable.tmp_happysmile,
-                description = "optionButton2")
+            MapButton()
```

Comment:
This should only be a temporary button because it doesn't make sense to have access to the map without any game set up. Maybe this should be annotated?